platform/ui/src/components/InputDateRange/InputDateRange.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRange, InputLabelWrapper } from '@ohif/ui';
+
+const InputDateRange = ({
+  label,
+  isSortable,
+  sortDirection,
+  onLabelClick,
+  value,
+  onChange,
+}) => {
+  const { startDate, endDate } = value;
+  return (
+    <InputLabelWrapper
+      label={label}
+      isSortable={isSortable}
+      sortDirection={sortDirection}
+      onLabelClick={onLabelClick}
+    >
+      <div className="relative">
+        <DateRange
+          startDate={startDate}
+          endDate={endDate}
+          onChange={({ startDate, endDate }) => {
+            if (onChange) {
+              onChange({ startDate, endDate });
+            }
+          }}
+        />
+      </div>
+    </InputLabelWrapper>
+  );
+};
+
+InputDateRange.defaultProps = {
+  value: {},
+};
+
+InputDateRange.propTypes = {
+  label: PropTypes.string.isRequired,
+  isSortable: PropTypes.bool.isRequired,
```

Comment:
`isSortable` is really necessary? it's only used to define if the sort icon should be displayed or not. Maybe we can use the `sortDirection` itself to define it.

- If `sortDirection` has any value, the field is automatically sortable, right?
- `sortDirection` can be "not required" -- field is not sortable if `sortDirection` is empty or  has`none` as value.

What do you think? Open to ideas.

Reply:
I'm okay with either option, so long as its documented.

I'm trying to think about a parent setting all as sortable or not. Does it make more sense to set this per input, or to clear sort direction? What would you expect as a consumer of this library?

Reply:
I have 2 thoughts here. 
>Maybe we can use the sortDirection itself to define it.

`isSortable + sortDirection` could be a single `sorting` with values as `['none', 'ascending', 'descending', 'hidden']` but the way its now, is more intuitive to understand the direction and if we show or not. 
> Does it make more sense to set this per input, or to clear sort direction?